ble/GapAdvertisingData.h

```diff
@@ -345,6 +431,27 @@ class GapAdvertisingData
         return (uint16_t)_appearance;
     }
 
+    /**
+     * Search advertisement data for field.
+     * Returns pointer to field if found, NULL otherwise.
```

Comment:
Could you make it explicit in the comment that the return value points to the `length` parameter of the matched ADV structure?